README.md

```diff
@@ -1 +1,100 @@
-# Rails_App_Template
\ No newline at end of file
+# YummyRecipes
+
+
+This project is made as part of the Microverse curriculum for module 5- RAILS AND JOB SEARCH PREPARATION
+. "YummyRecipes" is a food recipe blogs web application that allows users to manage collections of recipes.
+
+
+## Link
+
+[Loom Video]()
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Ruby
+- PostgreSQL
+- Ruby on Rails
+
+## `Getting Started`
+
+To get a local copy of this project:
+
+Clone this repository or download the Zip folder:
+```
+git clone https://github.com/HAMMAS-SALEEM/YummyRecipes.git
+```
+
+Enter in root directory:
+```
+cd YummyRecipes
+```
+Then run:
+```
+- bundle install
+- npm install
+```
+Create Database (Mandatory)
+
+```
+rails db:create
+rails db:schema:load
+```
+
+If you have made a migration the run this command
+
+```
+rails db:migrate
+```
+
+run ShareMe app on server:
+```
+rails s
+```
+
+Open Source code of ShareMe app on VS-Code
+
+```
+code .
+```
+Run tests
+
+```
+rspec
+```
+
+## Authors
+
+👤 **Hammas Bin Saleem**
+
+- GitHub: [@HAMMAS-SALEEM](https://github.com/HAMMAS-SALEEM)
+- Twitter: [@HammasSaleem4](https://twitter.com/HammasSaleem4)
+- LinkedIn: [HAMMAS SALEEM](https://www.linkedin.com/in/hammas-saleem)
+
+👤 **Trésor Sawasawa**
+
+- GitHub: [@tresorsawasawa](https://github.com/tresorsawasawa)
+- Twitter: [@tresorsawasawa](https://twitter.com/TresorSawasawa)
+- LinkedIn: [Trésor Sawasawa](https://www.linkedin.com/in/tresor-sawasawa/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ShahierNashaat/ruby-capstone/issues/new).
```

Comment:
- [ ] Kindly make sure you have a valid link to add a new issue. This will make your `README` more professional.

Reply:
Thanks. I have implemented the changes